snuba/datasets/discover.py

```diff
@@ -0,0 +1,301 @@
+                ("tags", Nested([("key", String()), ("value", String()), ])),
+                ("contexts", Nested([("key", String()), ("value", String()), ])),
+            ]
+        )
+
+        self.__events_columns = ColumnSet(
+            [
+                ("group_id", Nullable(UInt(64))),
+                ("primary_hash", Nullable(FixedString(32))),
+                ("type", Nullable(String())),
+                # Promoted tags
+                ("level", Nullable(String())),
+                ("logger", Nullable(String())),
+                ("server_name", Nullable(String())),
+                ("site", Nullable(String())),
+                ("url", Nullable(String())),
+                ("message", Nullable(String())),
+                ("search_message", Nullable(String())),
+                ("title", Nullable(String())),
+                ("location", Nullable(String())),
+                ("culprit", Nullable(String())),
+                ("received", Nullable(DateTime())),
+                ("geo_country_code", Nullable(String())),
+                ("geo_region", Nullable(String())),
+                ("geo_city", Nullable(String())),
+                ("sdk_name", Nullable(String())),
+                ("sdk_integrations", Nullable(Array(String()))),
+                ("sdk_version", Nullable(String())),
+                ("version", Nullable(String())),
+                ("http_method", Nullable(String())),
+                ("http_referer", Nullable(String())),
+                ("exception_stacks.type", Nullable(String())),
+                ("exception_stacks.value", Nullable(String())),
+                ("exception_stacks.mechanism_type", Nullable(String())),
+                ("exception_stacks.mechanism_handled", Nullable(UInt(8))),
+                ("exception_frames.abs_path", Nullable(String())),
+                ("exception_frames.filename", Nullable(String())),
+                ("exception_frames.package", Nullable(String())),
+                ("exception_frames.module", Nullable(String())),
+                ("exception_frames.function", Nullable(String())),
+                ("exception_frames.in_app", Nullable(UInt(8))),
+                ("exception_frames.colno", Nullable(UInt(32))),
+                ("exception_frames.lineno", Nullable(UInt(32))),
+                ("exception_frames.stack_level", Nullable(UInt(16))),
+                ("modules.name", String()),
+                ("modules.version", String()),
+            ]
+        )
+
+        self.__transactions_columns = ColumnSet(
+            [
+                ("trace_id", Nullable(UUID())),
+                ("span_id", Nullable(UInt(64))),
+                ("transaction_hash", Nullable(UInt(64))),
+                ("transaction_op", Nullable(String())),
+                ("start_ts", Nullable(DateTime())),
+                ("start_ms", Nullable(UInt(16))),
+                ("finish_ts", Nullable(DateTime())),
+                ("finish_ms", Nullable(UInt(16))),
```

Comment:
please let's have a TODO about cleaning this up once we have fixed how the timeseries works.